processor/src/services/payment.service.ts

```diff
@@ -0,0 +1,51 @@
+import { Cart } from '@commercetools/connect-payments-sdk';
+import { getCartById } from './../commercetools/cart.commercetools';
+import { convertCentsToEur } from '../utils/app.utils';
+import { MIN_CART_AMOUNT, MAX_CART_AMOUNT } from '../utils/constant.utils';
+import { readConfiguration } from '../utils/config.utils';
+import { GetOptionResponse } from '../types/payment.types';
+
+export const handleGetEasyCreditPaymentMethod = async (cartId: string): Promise<GetOptionResponse> => {
+  const cart: Cart = await getCartById(cartId);
+
+  const config = readConfiguration();
+
+  const result = {
+    webShopId: config.easyCredit.webShopId,
+  };
+
+  const errors: Array<{ code: string; message: string }> = [];
+
+  if (
+    cart?.billingAddress &&
```

Comment:
what if billingAddress or shippingAddress is missing?

Reply:
I added 2 more checks for those objects, please help me to have a look anh